chart/values.yaml

```diff
@@ -8,22 +8,68 @@ imagePullSecrets:
 # Use Self Signed Cert for TLS between private and public service GRPC
 mirrorServiceTLS: false
 
-fullService:
+image:
+  repository: mobilecoin/full-service-mirror
```

Comment:
Yes.  This is a self contained image with builds of full-service for the sub-module commit tested with the full-service-mirror binaries and the mc-ledger-validator binary (lives in the full-service repo).
The full-service image will remain and is currently independent of this repo.  This could of course change in the future but as of now it will build a copy of `full-service` & `mc-validator-service` for the current sub-module commit